routes/articleRoutes.js

```diff
@@ -71,6 +86,17 @@ router.route('/:slug')
     getArticle
   );
 
+router.route('/rating/:slug')
+  .patch(
+    verifyToken,
+    userIsMentor,
+    ensureArticleExists,
+    isRateValueSupplied,
+    inputTypeIsValid,
+    ratingIsInRange,
+    Rating.rateArticle
```

Comment:
Please destructure this up there too, to conform with what the others look like.